backend/typescript/services/implementations/authService.ts

```diff
@@ -181,7 +181,7 @@ class AuthService implements IAuthService {
         <title>Welcome Email</title>
       </head>
       <body>
-         <p><img src=https://i.ibb.co/txCj8db/drawer-logo.png
+         <p><img src=https://community-fridge-logo.s3.us-west-004.backblazeb2.com/community-fridge-logo.png
```

Comment:
is this the transparent image? it looks like it has a background for me

Reply:
Is it the grey background when you click the link? I think it shows up like that because the background of the circle is transparent. Here is what the image looks like in the emails for me (Outlook and Gmail)
<img width="830" alt="Screen Shot 2022-02-01 at 8 53 31 PM" src="https://user-images.githubusercontent.com/38363056/152081736-a2900346-6ad2-441d-b33f-6638168025a9.png">
![Screen Shot 2022-02-01 at 8 57 59 PM](https://user-images.githubusercontent.com/38363056/152081766-caa4fc2b-9a40-4ef4-a346-c7e5388c9cf7.png)